livekit_models.proto

```diff
@@ -99,6 +99,16 @@ message ParticipantInfo {
     // WS disconnected
     DISCONNECTED = 3;
   }
+  enum Type {
+    // normal participants, e.g. web clients
+    NORMAL = 0;
```

Comment:
"Normal" here may communicate that the other kinds are abnormal somehow. Would "standard" or "default" work better here?

Reply:
Agree, good idea!